flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/services/MesosServicesUtils.java

```diff
@@ -40,9 +41,11 @@ public static MesosServices createMesosServices(Configuration configuration) thr
 				return new StandaloneMesosServices();
 
 			case ZOOKEEPER:
-				final String zkMesosRootPath = configuration.getString(
+				final String zkMesosRootPath = ConfigurationUtil.getStringWithDeprecatedKeys(
+					configuration,
 					ConfigConstants.HA_ZOOKEEPER_MESOS_WORKERS_PATH,
-					ConfigConstants.DEFAULT_ZOOKEEPER_MESOS_WORKERS_PATH);
+					ConfigConstants.DEFAULT_ZOOKEEPER_MESOS_WORKERS_PATH,
+					ConfigConstants.ZOOKEEPER_MESOS_WORKERS_PATH);
```

Comment:
Thanks for the review. 

I looked into `HighAvailabilityOptions` and found that it doesn't contain all the HA options, some options are still in `ConfigConstants`. 
It looks like this part is still in the middle of refactoring, right? I also heard some discussion from @greghogan about generating configuration document from `ConfigOption` automatically. 
Is there any work I can help with, like move options from `ConfigConstants` to corresponding `xxConfigOptions` files or the automatic work? If it is the right direction and there is some work I can do, I would say to have another PR for the work, if not I would just move  `HA_ZOOKEEPER_MESOS_WORKERS_PATH` to `HighAvailabilityOptions` and finish this PR.